pubspec.yaml

```
@@ -28,6 +28,8 @@ dependencies:
   shared_preferences: ^0.5.6+3
   # Common Fonts
   google_fonts: ^0.5.0+1
+  # Toast
+  toast:
```

this plugin is better
also mention the current version always!
use `Pubspec Assist` extension in vscode to easily add packages
```suggestion
  fluttertoast: ^4.0.1
```